EventFilter/TotemRawToDigi/python/TotemVFATRawToDigi_cfi.py

```diff
@@ -0,0 +1,54 @@
+import FWCore.ParameterSet.Config as cms
+
+TotemVFATRawToDigi = cms.EDProducer("TotemVFATRawToDigi",
```

Comment:
define fillDescriptions method in TotemVFATRawToDigi so that framework can auto-generate the necessary configuration and also be able to check configuration as well as give help on parameters
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp